app/src/main/res/values/strings.xml

```diff
@@ -26,6 +26,15 @@
     <string name="number_converter_binary">Binary</string>
     <string name="number_converter_hexadecimal">Hexadecimal</string>
 
+    <!-- Screen / Tab Titles for Default Tab -->
+    <string name="menu_dices_value">dices</string>
```

Comment:
Needs `translatable="false"` for these strings.

Reply:
yep yep